tools/cluster_tools/mirror_image.py

```diff
@@ -149,6 +153,15 @@ def launch_node_commands(node, api_server, args):
     sys.stdout.flush()
     child.close()
 
+def send_pexpect_command(child, command, prompt):
```

Comment:
Please, don't create a new function while existing one makes this same.

If you want to decrease timeout to 60, use the existing one. You can reduce timeout if you wish there and include the message when the exception takes place.